### DISCUSSION TITLE
Personal Comment Area / Mood / Wisdom or something

### DISCUSSION THREAD

**Discussion by d0b3rmann** [Automattic/gravatar#71] [Ideas] [answered]
Hey there,

I'd like a personal influence taking part of my Profile in a personal Area that I can describe myself. Just another idea...

Regards

**Comment by aaronfc** [comment, marked as answer]
Hello @d0b3rmann you can write a personal description of yours in the "About Me" section as described in https://support.gravatar.com/your-profile/about-you/
